optimum/habana/diffusers/pipelines/pipeline_utils.py

```diff
@@ -370,6 +368,13 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
         bf16_full_eval = kwargs.get("torch_dtype", None) == torch.bfloat16
         kwargs["bf16_full_eval"] = bf16_full_eval
 
+        # Need to load custom ops lists before instantiating htcore
+        gaudi_config = GaudiConfig.from_pretrained(kwargs["gaudi_config"])
```

Comment:
@dsocek have you considered a situation where "gaudi_config" is not a key in kwargs? It seems in this function we already have `self.gaudi_config`. Please fix the logic. This creates an issue in one of the `test_textual_inversion` tests